docs/data/toolpad/getting-started/why-toolpad.md

```diff
@@ -17,15 +17,15 @@ After reviewing multiple approaches and tools, we noticed that there are a few r
 
 ### The problems with pro-code tools
```

Comment:
I'm not sure whether we need to talk about "problems with pro-code" as we talk about "problems with low-code". The way I see it is not that there are problems with pro-code. It's that for internal tooling there are different trade-offs you can make. e.g. trade in flexibility, and performance for speed of development and maintenance overhead.